MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.NumericUpDown.xaml

```diff
@@ -0,0 +1,58 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:po="http://schemas.microsoft.com/winfx/2006/xaml/presentation/options"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:converters="clr-namespace:MaterialDesignThemes.Wpf.Converters"
+                    xmlns:local="clr-namespace:MaterialDesignThemes.Wpf"
+                    xmlns:system="clr-namespace:System;assembly=mscorlib"
+                    xmlns:transitions="clr-namespace:MaterialDesignThemes.Wpf.Transitions"
+                    xmlns:wpf="clr-namespace:MaterialDesignThemes.Wpf">
+  <ResourceDictionary.MergedDictionaries>
+    <!--
+      we only bring in the dictionaries for controls which were "invented"
+      as part of Material Design in XAML Toolkit. Other themes, for existing controls
+      must be selected manually by the user, so that they override default styles.
+      The easiest way to do this is include:
+      
+      MaterialDesignTheme.Defaults.xaml
+      in your App.xaml
+    -->
+  </ResourceDictionary.MergedDictionaries>
+  <!-- set up default styles for our custom Material Design in XAML Toolkit controls -->
+  <Style TargetType="{x:Type wpf:NumericUpDown}">
```

Comment:
One idea for a cleaner look could be to match the look of the [one from MatBlazor](https://www.matblazor.com/NumericUpDownField)